bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -257,6 +261,11 @@ public String getPrefillService() {
         return prefillService;
     }
 
+    @JsonIgnore
```

Comment:
Please add these attributes in the interface. Can we change the property to "isHamburgerMenuEnabled"

Reply:
Done